### ISSUE TITLE
variables not expanded in @path strings

### ISSUE THREAD

**Issue by DavidBiesack** [sassoftware/unravl#4]
When referencing a `"@location"` to read a file from a URL/file, local variables are not expanded. thus, you can't do
```JavaScript
{
  "env" : { "path" : "tst/src/scripts/" },
  ...
  "assert": [
    { "json" : "@{path}expected.json"}
    ]
}    
```

**Comment by DavidBiesack** [comment]
The fix should also apply to UnravlRuntime which reads the initial script:
```JSON
[ "@{path}/file1.json", "@{path}/file1.json" ]
```

**Comment by DavidBiesack** [comment]
Fixed with pull request #5
